tests/all_mode/run.sh

```diff
@@ -101,8 +101,8 @@ function run() {
     # use sync_diff_inspector to check data now!
     check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
 
-    check_metric $WORKER1_PORT 'dm_syncer_replication_lag{task="test"}' 3 0 1
-    check_metric $WORKER2_PORT 'dm_syncer_replication_lag{task="test"}' 3 0 1
+    check_metric $WORKER1_PORT 'dm_syncer_replication_lag{task="test"}' 3 0 2
```

Comment:
why need to increase the `upper` to `2`? It seems that use `UTC_TIMESTAMP(6)` to replace `UTC_TIMESTAMP` when update ts will decrease the replication lag.

Reply:
I saw lag with 1.x (but < 1.5) in some cases. in the old `UTC_TIMESTAMP` it `==1`, but now it `> 1`.